samples/sample.yaml

```diff
@@ -26,5 +26,13 @@ reports:
       - name: "in-progress"
         config:
           report-on: "Epic"
-          wip-label-match: "(\\d+)-dev"   
+          wip-label-match: "(\\d+)-dev"
+      - name: "issue-count-cycle-time"
+        config:
+          feature-cycletime-limit: 200
```

Comment:
move these to the reports getDefaultConfiguration() - or at least put good defaults there and only add what needs to be overriden here.  Putting the in getDefaultConfiguration() also shows what settings are available for use.